libs/ui/src/lib/button/button.tsx

```diff
@@ -66,6 +67,7 @@ export const Button: React.FC<ButtonProps> = ({
   >
     {accessoryLeft && <div className="flex">{accessoryLeft}</div>}
     {children}
+    {accessoryRight && <div className="flex">{accessoryRight}</div>}
```

Comment:
you don't need the extra div here

- [ ] fix  `accessoryLeft`, `accessoryRight`
- [ ] take this example and update with `cva` (_do not monkey copy/paste, update according to the sizes we already have_)

```
  <button type="button" class="inline-flex items-center gap-x-1.5 rounded-md bg-indigo-600 px-2.5 py-1.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
    Button text
    <CheckCircleIcon class="-mr-0.5 h-5 w-5" aria-hidden="true" />
  </button>
  <button type="button" class="inline-flex items-center gap-x-1.5 rounded-md bg-indigo-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
    Button text
    <CheckCircleIcon class="-mr-0.5 h-5 w-5" aria-hidden="true" />
  </button>
  <button type="button" class="inline-flex items-center gap-x-2 rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
    Button text
    <CheckCircleIcon class="-mr-0.5 h-5 w-5" aria-hidden="true" />
  </button>
```

from https://tailwindui.com/components/application-ui/elements/buttons